Makefile

```
@@ -1,66 +1,63 @@
-
-html: pagefind
-	sphinx-build -M html . _build -j auto -n $(O) -Dhtml_extra_path=_redirects,_pagefind
-
-pagefind:
-	sphinx-build -M html . _build -j auto -n $(O)
-	mkdir -p _pagefind/pagefind
-	${PAGEFIND}
-
-live-html:	pagefind
-	sphinx-autobuild . _build -j auto -n $(O) --host 0.0.0.0 -Dhtml_extra_path=_redirects,_pagefind
+.PHONY: html clean live-html automations check-links anchors production convert-from-rst directories pagefind-binary netlify repo-data
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)